book/requirements.txt

```diff
@@ -1 +1 @@
-jupyter-book==0.12.1
+jupyter-book==0.12.3
```

Comment:
Seems that in the interim our Jupyter Book setup broke. I'm trying to debug things and then I'll build lock files for us to use in the future.